mlflow/models/python_api.py

```diff
@@ -75,6 +76,11 @@ def build_docker(
             The version of installed mlflow will be the same as the one used to invoke this command.
 
         mlflow_home: Path to local clone of MLflow project. Use for development only.
+
+        base_image: Base image for the Docker image. If not specified, the default image is either
+            UBUNTU_BASE_IMAGE = "ubuntu:20.04" or PYTHON_SLIM_BASE_IMAGE = "python:{version}-slim"
+            Note: If custom image is used, there are no guarantees that the image will work. You
+            may find greater compatibility by building your image on top of the ubuntu images.
```

Comment:
You should also inform user that if using custom image,  user must install `java` and `virtualenv` by themselves in the custom image otherwise MLflow building docker doesn't work